packages/hide/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "dist/index.js",
   "scripts": {
     "prepare": "npm run clean && npm run build",
-    "clean": "rm -rf dist && mkdir dist",
-    "build": "babel src/index.js -o dist/index.js"
+    "clean": "rm -rf dist",
+    "build": "babel src -d dist"
```

Comment:
Could you please revert this? It will conflict with my changes in this PR https://github.com/rebassjs/rebass/pull/515